### ISSUE TITLE
Typo in docs: initLocations should be inits?

### ISSUE THREAD

**Issue by jerry1970** [devvoh/parable#14]
From `Init\Example`:
```
 * the location of these scripts needs to be set in a Config file, using the root key
 * 'initLocations'. See app/Config/App.php.
```
From `Config\App`:
```
"inits" => [
    \Init\Example::class,
```

**Comment by devvoh** [comment]
Hah, yeah, I'll take this along!

**Comment by devvoh** [comment]
Fixed in 0.12.5!
